packages/media/src/types/mediaElement.ts

```diff
@@ -17,14 +17,16 @@
 /**
  * External dependencies
  */
-import PropTypes from 'prop-types';
+import type { DimensionableElement } from '@googleforcreators/units';
 
 /**
  * Internal dependencies
  */
-import { AnimationInputPropTypes } from '../types';
+import type { Resource } from './resource';
 
-export const PanEffectInputPropTypes = {
-  panDir: PropTypes.shape(AnimationInputPropTypes),
-  duration: PropTypes.shape(AnimationInputPropTypes),
-};
+export interface MediaElement extends DimensionableElement {
```

Comment:
Yeah, I was a bit conflicted about that. I didn't know where to put this. I don't want elements in the media package, but I also can't import from the elements package in animations. It's all because of the `getElementOffsets` util function.

This function has been moved around a lot lately - probably too much. Like some troubled teenager in and out of foster homes. I'm starting to think it feels abandoned 😢. Maybe it can find a safe and happy forever-home in the animations package <3

Reply:
Yep, if it's used only for `animations`, maybe it belongs there.

Reply:
Moved in 72c630baab.

Reply:
Hm, do we still need a separate `MediaElement` in the media package (and in the `elements` package)?